src/main.go

```diff
@@ -22,6 +22,7 @@ func dataHandler(w http.ResponseWriter, r *http.Request) {
 		http.Error(w, "Error encoding JSON", http.StatusInternalServerError)
 		return
 	}
+	//fmt.Println(analyze.AllResults)
```

Comment:
A useful thing is that git remembers everything. So commented sections of code can sometimes just be dropped; git has it, if you ever need it back!